job-service-db/src/main/resources/procedures/changelog.xml

```diff
@@ -85,6 +85,12 @@
                          schemaName="public">
         </createProcedure>
     </changeSet>
+    <changeSet id="create_drop_task_tables2_proc" runOnChange="true" author="Gamma Team">
+        <createProcedure path="procedures/dropTaskTables2.sql"
```

Comment:
How about `dropTaskTablesSP.sql` instead of `dropTaskTables2.sql`?  To prevent it sounding like just another version.